File: frontend/src/models/Link.ts
export class Link {
  public id: number;
  public createdAt: string;
  public url: string;
  constructor(id: number, url: string, createdAt: string) {
    this.id = id;
    this.url = url;
    this.createdAt = createdAt;
  }
}

File: WebsitePerformanceEvaluator.Domain/Models/Link.cs
namespace WebsitePerformanceEvaluator.Domain.Models;

public class Link
{
    public Link()
    {
        LinkPerformances = new HashSet<LinkPerformance>();
    }

    public int Id { get; set; }
    public DateTime CreatedAt { get; set; }
    public string Url { get; set; }
    public virtual ICollection<LinkPerformance> LinkPerformances { get; set; }
}